File: Webhacking.kr/prob33/9.py
import requests

url = 'http://webhacking.kr/challenge/bonus/bonus-6/nextt.php'
cookies = {
	'PHPSESSID': ''
}
payloads = {
	'ans': ''
}

answer = ''
i = 97
while i <= 122:
	answer += chr(i)
	i += 2

payloads['ans'] = answer

res = requests.get(url, cookies=cookies, params=payloads)
print(res.text)

File: Webhacking.kr/prob33/9.php
<?php
for($i=97;$i<=122;$i=$i+2)
{
$ch=chr($i);

$answer.=$ch;

}

if($_GET[ans]==$answer)
{
echo("<a href=###>Next</a>");
}
else
{
echo("Wrong");
}
?>